export const cusotomError = (errMessage: string) => {
  throw new Error(errMessage);
  return;
};
